static/app/utils/useCommitters.tsx

```diff
@@ -27,6 +27,7 @@ function useCommitters(
     makeCommittersQueryKey(org.slug, projectSlug, eventId),
     {
       staleTime: Infinity,
+      retry: false,
```

Comment:
Should we make this the default maybe? cc @malwilley

Reply:
I think we can, yeah. Our API is very stable so retries might not be providing any value for us.

However, there is a separate issue here in that this will make `isError = true`. The components using `useCommitters` don't use that, but in most cases you'd want to check for that and render an error message. So we probably want some sort of `allowedErrorStatuses` or something. Or you could pass a custom query function that ignored 404s and returned null in that case. But if this is a common pattern (where 404 should not be considered an error) then we probably want to put something in `queryClient`